.github/workflows/build-push-docker.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker build and push to ghcr.io/ai-cfia and lint test
+
+on: 
+  pull_request:
+    types:
+      - opened
+      - closed
+      - synchronize
+
+jobs:
+  lint-test:
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-lint-test-python.yml@main
+    secrets: inherit
+
+  deploy:
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-build-push-container-github-registry.yml@63-as-a-devops-i-would-like-to-create-a-workflow-to-push-images-to-this-organisation-docker-registry
```

Comment:
Once the branch 63 is merged, let's make sure this points to the main branch.

Reply:
@k-allagbe since this is a future todo, should probably be a comment instead of a "request for changes" that blocks merging.

Reply:
Okay got it. Are we okay merging with the unmerged workflow then?

Reply:
IMO yes because having a pionter to the branch makes it clear that the called workflow is under development but OTOH we want to minimize blocking for deployment